std/traits.d

```diff
@@ -5036,21 +5036,21 @@ template ImplicitConversionTargets(T)
                        float, double, real, char, wchar, dchar);
     else static if (is(T == byte))
         alias ImplicitConversionTargets =
-            AliasSeq!(short, ushort, int, uint, long, ulong, CentTypeList,
```

Comment:
This makes sense, but ends up lying, since this compiles:

```d
int i = -5;
ulong oops = i;
```

Reply:
`ImplicitConversionTargets` already "lies" like this in several cases, by design. The issue is that it lies about *some* signed → unsigned conversions (e.g. `int` → `uint`) but not about others (e.g. `int` → `ulong`).

Reply:
That makes sense, but I'm not sure we should be *adding* to the list of lies.

Reply:
> That makes sense, but I'm not sure we should be _adding_ to the list of lies.

The alternative would be, to alter the documentation and accept `int` → `uint` too. Might be easier with regard to the deprecation discussion. I've got no opinion on my own here. My motivation is to handle an issue "somehow". (I used a random generator, which selected this issue.)

Reply:
An other alternative would be to add `SaveConversionTargets` and `AllConversionTargets` and deprecate this function altogether...